src/localconf.js

```diff
@@ -24,6 +24,7 @@ function buildConfig(opts) {
     fenceURL: process.env.FENCE_URL,
     indexdURL: process.env.INDEXD_URL,
     cohortMiddlewareURL: process.env.COHORT_MIDDLEWARE_URL,
+    gwasWorkflowURL: process.env.GWAS_WORKFLOW_URL,
```

Comment:
Should these VA specific things be in this common .js file? 
And do we have a scenario where this URL is set to something different than the default value (which seems to be `${hostname}ga4gh/wes/v2/` (some lines below)?
Same questions apply to `cohortMiddlewareURL`.

Reply:
I'm not sure what an alternative solution would be for having all the urls in sync for qa/dev/prod/etc without making more changes for api endpoint base url

Reply:
are these relative urls not the same for all these environments?

Reply:
no, the relative urls are prefaced the same because they use `hostname` so i think removing this would break it

Reply:
oh you're saying they should share `hostname` instead of being declared separately

Reply:
this will be addressed in separate ticket

Reply:
https://ctds-planx.atlassian.net/browse/VADC-111